recipes-support/provisioning/files/provision-tpm.sh

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env bash
+#===============================================================================
+#
+#  provision-tpm.sh
+#
+#  Copyright (C) 2021 by PHYTEC Messtechnik GmbH
+#  All rights reserved.
+#
+#  This program is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU General Public License version 2 as published by
+#  the Free Software Foundation.
+#
+#===============================================================================
+
+printf "======================================\n" > /dev/console
+printf "PHYTEC TPM2 Test Provision Version 0.2\n" > /dev/console
+printf "======================================\n" > /dev/console
+
+USEHSM=1
+CA_P12=""
+CA_ROOT=""
+HSMCERT_LABEL=""
+HSM_USER_PIN=""
+while getopts 'ht:l:r:P:' arg ; do
+    case "$arg" in
+    h)  printf "h  Show this help \n" > /dev/console
+        printf "t  Path to the PKCS#12 container with CA and private Key\n" > /dev/console
+        printf "l  Label for Cert and Key in HSM\n" > /dev/console
+        printf "P  HSM User Pin\n" > /dev/console
+        printf "r  Path to Root CA\n" > /dev/console
```

Comment:
Maybe we can add a `--clear` option to clear the TPM and any generated metadata.

Reply:
See https://github.com/mschwan-phytec/meta-esec-iotdm/commit/a33215e34ef37d9961d865f645b4f9409dd92a41.

Reply:
clear is integrated